system/jlib/jptree.cpp

```diff
@@ -5367,6 +5367,21 @@ IPropertyTree *createPTreeFromXMLString(unsigned len, const char *xml, byte flag
 //////////////////////////
 /////////////////////////
 
+inline bool checkSanitizedHide(const char *val)
```

Comment:
picky: This name doesn't really describe the function rather compared with something like isHiddenWhenSanitized()

Similarly isSanitizedAndHidden() instead of checkSanitizeFlagsAndHide.  (Being very picky checkSanitizeFlagsAndHide implies it is performing an action, rather than testing a condition.)

Reply:
ok, noted, I'll rebase as is for now though.